docs/topics/sealed-classes.md

```diff
@@ -50,6 +50,18 @@ Subclasses of sealed classes must have a proper qualified name. They can't be lo
 >
 {type="note"}
 
+### Inheritance in multiplatform projects
+
+In [multiplatform projects](mpp-intro.md), there comes one more restriction: direct subclasses of sealed classes must
```

Comment:
I prefer more simplified wording in such cases, like "There is an inheritance restriction in multiplatform projects:", but yours is also good!